tests/standalone_tests/conftest.py

```diff
@@ -556,6 +559,27 @@ def localnodetmp_db_cursor():
     return _create_db_cursor(MONETDB_LOCALNODETMP_PORT)
 
 
+def insert_data_to_db(
+    table_name: str,
+    table_values: List[List[Union[str, int, float]]],
+    db_port,
```

Comment:
I would suggest to provide the cursor in the method instead of the port, so that we don't potentially create many cursors if we use this method many times.

Reply:
done